### PR TITLE
Add new card design to pools

### PR DESCRIPTION
**Pull request by kattylucy**
- Add card design to pools landing page
- Update theme with new colors theme (https://www.figma.com/design/ng7qdNcSCXSDA6ZUdWIs6u/Pool-Overview%2F-Pool-Detail?node-id=2873-1095#935009325)

#2391 

- [ ] Dev

### CODE REVIEW

File: centrifuge-app/src/components/PoolCard/index.tsx [hunk 1]
```diff
@@ -32,100 +67,116 @@ export function PoolCard({
...
-                />
-              ) : (
-                '—'
-              )}
-            </Ellipsis>
-            {status === 'Upcoming' && apr ? <Text variant="body3"> target</Text> : ''}
-          </TextWithPlaceholder>
+  return (
+    <Card
+      marginRight={20}
```
Comment (sophialittlejohn) [inline centrifuge-app/src/components/PoolCard/index.tsx:105]: this should be marked as an anchor tag for better semantics HTML. Should be as simple as `as={a}`, you'll probably still have to remove the default styling on the a tag.

File: centrifuge-app/src/components/PoolCard/index.tsx [hunk 2]
```diff
@@ -32,100 +67,116 @@ export function PoolCard({
...
+          <Text as="span" variant="body3" color="textButtonPrimaryDisabled">
+            APY
+          </Text>
+          {tranchesData?.map((tranche) => renderText(`${tranche.apr}`))}
+        </Stack>
+        <Stack>
+          <Text as="span" variant="body3" color="textButtonPrimaryDisabled">
+            Min Investment
+          </Text>
+          {tranchesData?.map((tranche) => renderText(`${tranche.minInvestment}`))}
```
Comment (sophialittlejohn) [inline centrifuge-app/src/components/PoolCard/index.tsx:162]: This looked too big in the preview, I think you need to convert it to decimal first

File: centrifuge-app/src/components/PoolCard/index.tsx [hunk 3]
```diff
@@ -32,100 +72,118 @@ export function PoolCard({
...
-                '—'
-              )}
-            </Ellipsis>
-            {status === 'Upcoming' && apr ? <Text variant="body3"> target</Text> : ''}
-          </TextWithPlaceholder>
+  return (
+    <StyledCard
+      marginRight={20}
+      marginBottom={20}
+      onClick={() => navigate(`/pools/${poolId}`)}
```
Comment (onnovisser) [inline centrifuge-app/src/components/PoolCard/index.tsx:114]: Better would be to use a React Router Link instead. That'll render is as an `a` too, but with an `href` for better usability.

**Review by sophialittlejohn — APPROVED** [review]
LGTM 🔥

**Pull request merged**